forc/src/ops/forc_init.rs

```diff
@@ -52,6 +53,40 @@ struct ContentResponse {
     url: String,
 }
 
+fn print_welcome_message() {
+    let read_the_docs = format!(
+        "Read the Docs:\n- {}\n- {}\n- {}",
+        Link::new("Sway Book", "https://fuellabs.github.io/sway/latest/"),
+        Link::new(
+            "Rust SDK Book",
+            "https://fuellabs.github.io/fuels-rs/latest/index.html"
+        ),
+        Link::new(
+            "Typescript SDK Docs",
+            "https://github.com/FuelLabs/fuels-ts"
+        )
+    );
+
+    let join_the_community = format!(
+        "Join the Community:\n- Follow us {}
+- Ask questions in dev-chat on {}",
+        Link::new("@SwayLang", "https://twitter.com/SwayLang"),
+        Link::new("Discord", "https://discord.com/invite/xfpK4Pe")
+    );
+
+    let report_bugs = format!(
+        "Report Bugs:\n- {}",
+        Link::new("Sway Issues", "https://github.com/FuelLabs/sway/issues/new")
```

Comment:
These links aren't working for me in GNOME's Terminal or Console apps. Do you mind if we remove the `terminal-link` crate and include the links directly? E.g. perhaps something like:

```
Read the Docs:
- Sway Book: https://fuellabs.github.io/sway/latest
- Rust SDK Book: https://fuellabs.github.io/fuels-rs/latest
...
```
and so on?

Reply:
Weird, the links work on my terminal! But for safety, sure we can include the links directly.